applications/CompressiblePotentialFlowApplication/python_scripts/define_wake_process_2d.py

```diff
@@ -51,9 +51,16 @@ def ExecuteInitialize(self):
 
```

Comment:
Is this file intentionally changed in this PR?

Reply:
Trailing_edge and wake_elements model parts and lists had to be added in order to make the fsi simulation run under __init__ 
appending the elements inside the lists had to be added as well inside __MarkWakeElements
and the cleaning part was added long time ago. I think it is better to leave it in case someone needs to check its behavior in the python code, even though it is not called in the coupled solver anymore